### DISCUSSION TITLE
Building Variant

### DISCUSSION THREAD

**Discussion by curlerhaus** [guysoft/FullPageOS#560] [Q&A]
I'm trying to build a variant using the directions listed below from your readme. Everything seems to work except the last command. Are these directions still correct? 

sudo apt install coreutils p7zip-full qemu-user-static

git clone https://github.com/guysoft/CustomPiOS.git
git clone https://github.com/guysoft/FullPageOS.git
cd FullPageOS/src/image
wget -c --trust-server-names 'https://downloads.raspberrypi.org/raspios_lite_armhf_latest'
cd ..
../../CustomPiOS/src/update-custompios-paths
sudo modprobe loop
sudo bash -x ./build_dist

**Comment by guysoft** [comment]
I warmly suggest you use the docker build method, it tends to be more stable.

The last command is the actual build command. 
If you want to build a variant you should use:

```
 sudo bash -x ./build_dist [VARIANT NAME]
 ```
 As explained here:
 https://github.com/guysoft/FullPageOS?tab=readme-ov-file#building-fullpageos-variants
 
 But I really suggest you use:
 https://github.com/guysoft/CustomPiOS/wiki/Building-with-Docker#docker-run
